### PR TITLE
Branch- Cart page 

### PR DESCRIPTION
**Pull request by tanishk15sharma**
**CART-MANAGEMENT-Page**

- Item  card with price details and  2 buttons 1) Remove from cart  2 )Add to wish list .  
- TOTAL PRICE -  summing up all the cart items with delivery charges and discount 
Cart-managment
.

### CODE REVIEW

File: pages/cart.css [hunk 1]
```diff
@@ -0,0 +1,121 @@
+@import url("https://fonts.googleapis.com/css2?family=Comfortaa:wght@700&family=Quicksand:wght@300&family=Roboto:wght@100;300;400;500;700&display=swap");
+
+.cart-container {
+  display: flex;
+  justify-content: center;
+}
+.cart-item {
+  display: flex;
+  gap: 1rem;
+  /* border: 1px solid red; */
```
Comment (KanekarSnehal) [inline pages/cart.css:10]: remove comments

File: pages/cart.css [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  height: 100%;
+  object-fit: cover;
+}
+.card-content {
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  gap: 0.5rem;
+  width: 80%;
+  /* border: 1px solid red; */
```
Comment (KanekarSnehal) [inline pages/cart.css:29]: remove comments

File: pages/cart.css [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+.product-price span:nth-child(2) {
+  font-size: 0.8rem;
+  text-decoration: line-through;
+  color: var(--secondary-color);
+}
+
+.product-price span:nth-child(3) {
+  font-size: 0.8rem;
+  color: #ff715b;
```
Comment (KanekarSnehal) [inline pages/cart.css:50]: Suggestion: If possible use CSS variables.

File: pages/cart.css [hunk 4]
```diff
@@ -64,3 +68,54 @@
...
+  background-color: #fdf4f3;
+
+  margin: 1rem;
+  padding: 1rem;
+}
+.bold {
+  font-weight: 600;
+  font-size: 1.2rem;
+}
+hr {
```
Comment (sharathnair9999) [inline pages/cart.css:82]: try to reduce such short code by making utility classes. would be a good practice

File: pages/cart.html [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+        <div class="nav-icon-div">
+          <img src="../assets/nav-img/heart.svg" alt="wishlist-icon" />
+          <span>Wishlist</span>
+        </div>
+      </div>
+    </nav>
+
+    <main class="cart">
+      <!-- <div class="container"> -->
+      <!-- <h3 class="h3 ta-center mb-6">Cart (1)</h3> -->
```
Comment (KanekarSnehal) [inline pages/cart.html:43]: remove comments

File: pages/cart.css [hunk 6]
```diff
@@ -64,3 +68,54 @@
...
+  html {
+    font-size: 90%;
+  }
+}
+@media (max-width: 660px) {
+  html {
+    font-size: 85%;
+  }
+  .cart-container {
+    flex-direction: column;
```
Comment (sharathnair9999) [inline pages/cart.css:119]: Even here you can follow utility classes

**Review by sharathnair9999 — COMMENTED** [review]
Rest of the code looks fine.

File: pages/cart.html [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+            </li>
+            <hr />
+          </ul>
+          <p class="discount-txt">You will save &#8377; 1000 on this order</p>
+          <button class="btn primary-btn">PLACE ORDER</button>
+        </div>
+      </div>
+    </main>
+
+    <script
```
Comment (MaruthiPothuganti) [inline pages/cart.html:120]: You can place the script tag of fontawesome in the head. As the icons will not load until the whole page is parsed.

File: pages/cart.css (hunk 4 above)
Comment (tanishk15sharma) [reply pages/cart.css:82]: yes you are right

File: pages/cart.html [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+          <img src="../assets/nav-img/heart.svg" alt="wishlist-icon" />
+          <span>Wishlist</span>
+        </div>
+      </div>
+    </nav>
+
+    <main class="cart">
+      <!-- <div class="container"> -->
+      <!-- <h3 class="h3 ta-center mb-6">Cart (1)</h3> -->
+
```
Comment (MaruthiPothuganti) [inline pages/cart.html:44]: if these arent of any use, You can delete them

**Review by KanekarSnehal — APPROVED** [review]
Rest all looks good and well structured.

File: pages/cart.html (hunk 5 above)
Comment (tanishk15sharma) [reply pages/cart.html:43]: sure

File: pages/cart.html [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+                <span>Rs. 2999</span>
+                <span>(40% OFF)</span>
+              </div>
+              <p>
+                Quantity: <i class="fa-solid fa-circle-minus"></i>
+                1
+                <i class="fa-solid fa-circle-plus"></i>
+              </p>
+              <button class="btn">Remove from Cart</button>
+              <button class="btn">Move to wishlist</button>
```
Comment (MaruthiPothuganti) [inline pages/cart.html:92]: Always have one Main CTA/ Button (Primary Button) beside an outline button as that differentiates the purpose of a button

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggestes some changes. Everything else looks good

File: pages/cart.html (hunk 7 above)
Comment (tanishk15sharma) [reply pages/cart.html:120]: thank you for this info

**Pull request merged**